XMLtoMD/GuidelineXmlToMD/MarkdownOut/MdExtensions.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace MarkdownOut {
```

Comment:
Braces should be put on their own line. The [.editorconfig](https://github.com/IntelliTect/CodingGuidelines/blob/master/.editorconfig#L88) should be flagging these. If it isn't it could be because it is a level higher than the solution.